mirrord/intproxy/src/lib.rs

```diff
@@ -200,6 +212,26 @@ impl IntProxy {
         !self.task_txs.layers.is_empty()
     }
 
+    /// Logs information about currently connected processes
+    fn log_connected_processes(&self) {
+        let count = self.connected_processes.len();
+        if count == 0 {
+            return;
+        }
+
+        tracing::info!("Connected processes ({}): ", count);
+        for (layer_id, process_info) in &self.connected_processes {
+            tracing::info!(
+                layer_id = layer_id.0,
+                pid = process_info.pid,
+                name = %process_info.name,
+                cmdline = ?process_info.cmdline,
+                loaded = process_info.loaded,
+                "Process connected"
```

Comment:
Could be a bit confusing, cause "Process connected" sounds like we're announcing that event just happened. How about:
```suggestion
                "connected processes info"
```